src/lwc/geFormFieldLookup/geFormFieldLookup.js

```diff
@@ -204,7 +205,12 @@ export default class GeFormFieldLookup extends LightningElement {
      * @returns {Promise<void>}
      */
     retrieveLookupOptions = async (searchValue, sObjectType) => {
-        this.options = await doSearch({searchValue, sObjectType});
+        if (sObjectType === 'RecordType') {
```

Comment:
Could we import this from @salesforce/schema/RecordType? I'm not sure it's critical since the core object name is pretty locked, but the string literal is avoided at least